imagedephi/redact/redact.py

```diff
@@ -143,7 +158,21 @@ def redact_images(
                 continue
             if not redaction_plan.is_comprehensive():
                 logger.info(f"Redaction could not be performed for {image_file.name}.")
+                failed_file = failed_dir / image_file.name
+                # Using copy2 preserves metadata
+                # https://docs.python.org/3/library/shutil.html#shutil.copy2
+                copy2(image_file, failed_file)
```

Comment:
Should we _try_ to hardlink this and only copy it if we can't?

Reply:
ah, yes that would probably be best

Reply:
added in 5854456d39e1f95ba6f24d220f4f1a7a4c4be540